flink-runtime-web/web-dashboard/app/partials/jobs/job.jade

```diff
@@ -43,6 +43,10 @@ nav.navbar.navbar-default.navbar-fixed-top.navbar-main(ng-if="job")
     span.navbar-info-button.btn.btn-default(ng-click="cancelJob($event)")
       | Cancel
 
+  .navbar-info.last.first(ng-if!="job.type=='STREAMING' && (job.state=='RUNNING' || job.state=='CREATED')")
```

Comment:
Not sure exactly any more... I just tried to reproduce my search... I guess it was one of this:
http://jade-lang.com/reference/code/
https://stackoverflow.com/questions/6926247/nodejs-jade-escape-markup